trio/_core/_io_epoll.py

```diff
@@ -184,8 +185,10 @@ class EpollWaiters:
 class EpollIOManager:
     _epoll = attr.ib(factory=select.epoll)
     # {fd: EpollWaiters}
-    _registered = attr.ib(factory=lambda: defaultdict(EpollWaiters))
-    _force_wakeup = attr.ib(factory=WakeupSocketpair)
+    _registered = attr.ib(
+        factory=lambda: defaultdict(EpollWaiters), type=DefaultDict[int, EpollWaiters]
+    )
+    _force_wakeup = attr.ib(factory=WakeupSocketpair, type=WakeupSocketpair)
```

Comment:
Do you need the type= here? I would expect mypy to be able to infer it from the factory return type.

Reply:
You're right, that was not needed! Thanks, fixed.